spec/factories/users.rb

```diff
@@ -1,6 +1,14 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:email_address, User.count) {|n| "liono#{n}@some-cartoon-show.com" }
+    # don't freak out if the database isn't created yet
+    # http://stackoverflow.com/a/27745684/358804
```

Comment:
I figured you were trying to trigger something which would create the database. If this is just to provide a default, we should figure out what's really causing the error and resolve it.